### PR TITLE
feat(ios): resource file validation before use

### PR DESCRIPTION
**Pull request by jahorton**
Requires #5153 as presently written.

Fixes #5178.

I'm not immediately sure on how to properly user test all of the error handling, given that the trigger occurs behind multiple other guards during the resource installation process.  This _is_ meant to guard against a rather elusive error that seems to be occurring, after all.

That said, I was able to test the 'fallback' functionality by careful use of the debugger and deleting the backing .js file at the correct time.  The default keyboard (re-)installed and loaded successfully.

Sadly... I'm not immediately clear on if and where to produce an alert should a keyboard-loading error occur; while it's not difficult to do so in-app, it's a different story for the system keyboard.  So... the reset behavior may be a bit "obtuse" to end users, though at least we'll get Sentry logs about it whenever it does happen.  If the user enables error reporting, of course.

### CODE REVIEW

**Comment by darcywong00** [comment]
Did you mean to tag #5073? That's just incrementing master

File: ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift [hunk 1]
```diff
@@ -315,12 +330,24 @@ extension KeymanWebViewController {
...
     let stub: [String: Any] = [
       "id": lexicalModel.id,
       "languages": [lexicalModel.languageID], // Change when InstallableLexicalModel is updated to store an array
-      "path": storage.lexicalModelURL(for: lexicalModel).absoluteString
+      "path": fileURL.absoluteString
     ]
+
+    guard FileManager.default.fileExists(atPath: fileURL.path) else {
+      let event = Sentry.Event(level: .error)
+      event.message = SentryMessage(formatted: "File missing for keyboard")
```
Comment (darcywong00) [inline ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift:343]: Isn't this block for lexicalModel instead of keyboard?

**Comment by jahorton** [comment]
> Did you mean to tag #5073? That's just incrementing master

I was bound to mix up the numbers at least once eventually.  #5076.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.59-alpha](https://keyman.com/downloads/releases/alpha/15.0.59)

**Pull request merged**